Content.Server/Power/Components/PowerMonitoringConsoleComponent.cs

```diff
@@ -0,0 +1,17 @@
+#nullable enable
```

Comment:
```suggestion
```

Reply:
As I said, ported old test code...
Anyway fixed in ac07a2b5d932ee335d2eb99c6105a3eec0024cb6